iceoryx_hoofs/test/moduletests/test_posix_mutex.cpp

```diff
@@ -46,40 +46,62 @@ class Mutex_test : public Test
         }
     }
 
-    iox::posix::mutex sutNonRecursive{false};
-    iox::posix::mutex sutRecursive{true};
+    void signalThreadReady()
+    {
+        m_waitForThread.store(false);
+    }
+
+    void waitForThread()
+    {
+        while (m_waitForThread.load())
+        {
+        }
+    }
+
+    template <typename T>
+    int64_t getDuration(const T& start, const T& end)
+    {
+        return std::chrono::duration_cast<std::chrono::nanoseconds>(end - start).count();
+    }
+
+    static constexpr int64_t WAIT_IN_MS = 100;
+    static constexpr int64_t WAIT_IN_NS = WAIT_IN_MS * 1000000;
```

Comment:
This is only used in one test. Could it be moved there?
Also, 100ms are quite much. If the mutex is not locked, it takes essentially no time between the two measurements